MdeModulePkg/Bus/Pci/SdMmcPciHcDxe/SdMmcPciHci.h

```diff
@@ -478,6 +481,30 @@ SdMmcHcStopClock (
   IN UINT8                Slot
   );
 
+/**

```

Comment:
@aimanrosli23 
Please ensure you do not revert the changes made in commit:
SHA-1: 49accdedf956f175041040e677163b7cbb746283
* MdeModulePkg/SdMmcPciHcDxe: Hook SwitchClockFreq after SD clock start